BedrockServer.cpp

```diff
@@ -3,6 +3,19 @@
 #include "BedrockServer.h"
 #include "BedrockPlugin.h"
 
+// Status is special - it has commands that need to be handled by the sync node.
+#include <plugins/Status.h>
+
+// Definitions of static variables.
+condition_variable BedrockServer::_syncThreadReadyCondition;
+mutex BedrockServer::_syncThreadInitMutex;
+bool BedrockServer::_syncThreadReady = false;
```

Comment:
This is . going to go away in the refactor entirely, so I'm not going to bother with it now.